### ISSUE TITLE
Youtube Video Downloader GUI

### ISSUE THREAD

**Issue by bislara** [bislara/code-n-stitch#5]
## Description 

Make a GUI for downloading videos from Youtube.

- Take input as the URL of the video
- Show all the details like title, rating, view count, author, length of the video, description, likes, dislikes, etc
- Show available video resolution types to download.
- Give the option to user to select the video type to download.
- Get the location where the user wants to store the video.

## Technologies and Languages

- [ ] Python 
- [ ] Pafy Library 
- [ ] PyQt5(Recommended) 
- [ ] Tkinter

**Comment by Arpita-25** [comment]
Hi! I would like to work on this!

**Comment by iriyagupta** [comment]
Hi, I would like to work on this. I am done with the code, trying to create a GUI

**Comment by bislara** [comment]
On the first-come basis, I am assigning this to @iriyagupta. You can go for it. 👍

**Comment by bislara** [comment]
Any update? Feel free to reach out to me if you need any help.

**Comment by bislara** [comment]
@iriyagupta  Any progress?

**Comment by bislara** [comment]
@iriyagupta  has been unassigned from the issue due to the absence of response since the last 24 hours, making this issue up for grabs for anyone interested in it.

**Comment by bislara** [comment]
@Arpita-25  Would you like to take this now?

**Comment by Arpita-25** [comment]
Yes I would like to work on it...GUI part is completed... working on the
back end operation

**Comment by Arpita-25** [comment]
@bislara yes i would take it

**Comment by bislara** [comment]
Okay cool 👍🤟
